README.md

```diff
@@ -104,10 +104,75 @@ To get specific instructions for your preferred shell run:
 mongocli completion <bash|zsh|fish|powershell> --help
 ```
 
+## AtlasCLI (Pre-Release)
+![GO tests](https://github.com/mongodb/mongocli/workflows/GO%20tests/badge.svg)
+![golangci-lint](https://github.com/mongodb/mongocli/workflows/golangci-lint/badge.svg)
+
+The MongoDB AtlasCLI is a modern command line interface that enables you to manage MongoDB Atlas from the terminal.
+
+![atlascli-atlas-quickstart](https://user-images.githubusercontent.com/5663078/156184669-57c8ddce-6f0a-4e84-9311-2d996cb27942.gif)
+
+## Installing
+
+AtlasCLI is currently in the pre-release phase, so it should not be used in production environment. 
+
+### Pre-built Binaries
+
+Download the appropriate version for your platform from [AtlasCLI releases](https://github.com/mongodb/mongocli/releases).
+Once downloaded, the binary can be run from anywhere.
+You don't need to install it into a global location.
+This works well for shared hosts and other systems where you don't have a privileged account.
+
+Ideally, you should place this binary somewhere in your `PATH` for easy use.
+`/usr/local/bin` is the most probable location.
+
+### Build From Source
+
+#### Prerequisite Tools
+- [Git](https://git-scm.com/)
+- [Go (at least Go 1.17)](https://golang.org/dl/)
+
+#### Fetch Source
+
+```bash
+git clone https://github.com/mongodb/mongocli.git
+cd mongocli
+```
+
+#### Build
+
+To build `atlascli`, run:
+
+```bash
+make build-atlascli
+```
+
+The resulting `atlas` binary is placed in `./bin`.
+
+#### Install
+
+To install the `atlas` binary in `$GOPATH/bin`, run:
+
+```bash
+make install-atlascli
+```
+
+**Note:** running `make build-atlascli` is not needed when running `make install-atlascli`.
+
+
+## Usage
+
+Run `atlas help` for a list of available commands
+or check our online documentation for more details.
+
+### Configuring AtlasCLI
+To use AtlasCLI you'll need to run `atlas auth login` and follow the prompted steps on your terminal.
```

Comment:
Here and everywhere, use this structure:
To do X, run

This means, remove "you'll need to"